packages/engine/src/initializeEngine.ts

```diff
@@ -54,6 +55,10 @@ export const setupEngineActionSystems = () => {
     type: SystemUpdateType.FIXED_LATE,
     systemFunction: OutgoingActionSystem
   })
+  initSystemSync(world, {
```

Comment:
system should not be initialized like this, these are core internal engine systems, should be initialized in `initializeCoreSystems` asynchronously instead

Reply:
Thanks, moved.